helper/restore_helper.go

```diff
@@ -239,6 +243,10 @@ func doRestoreAgent() error {
 			}
 		}
 
+		if tableOid == skipOid {
+			goto LoopEnd
```

Comment:
let's add some log message in case we skip iterations.

Reply:
> let's add some log message in case we skip iterations.

Done